app/jobs/tags/bust_cache_job.rb

```diff
@@ -2,8 +2,11 @@ module Tags
   class BustCacheJob < ApplicationJob
     queue_as :tags_bust_cache
 
-    def perform(name, cache_buster = CacheBuster)
-      cache_buster.bust_tag(name)
+    def perform(tag_name)
+      tag = Tag.find_by(name: tag_name)
```

Comment:
If we can't find a tag, should we `raise` or surface that error? Are we okay with just `return`ing out of here and not knowing that we tried to run the `BustCacheJob` on a tag that doesn't exist?

Reply:
Your point makes absolute sense, but we kinda decided to first move over all the jobs to sidekiq workers and then figure out which ones we want to surface the errors from. Some can fail silently, some shouldn't. Keep in mind that retried jobs will eventually land in the dead queue if they don't work.

There also some error handling techniques builtin in Sidekiq we might want to employ: https://github.com/mperham/sidekiq/wiki/Error-Handling

ps.  I plan on writing an overview of our caching system, because I don't fully know how it works either, I'll definitely ask for your review :D

Reply:
@rhymes that totally makes sense! I'll try to keep an eye out for when we shift gears from "shift over workers" to "figure out the errors in them".

Also looking forward to learning about our caching system from you!! 😆